### DISCUSSION TITLE
Equivalent of Airflow KubernetesPodOperator with a pre-built Docker image

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17509] [Q&A] [answered]
Hi team,

In Airflow, kubernetespodoperator can be used for running a pre-built docker image, is there any similar feature in Dagster?
<https://airflow.apache.org/docs/apache-airflow-providers-cncf-kubernetes/stable/operators.html#kubernetespodoperator>

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1698722302926129?thread_ts=1698722302.926129&cid=C01U954MEER).*

**Comment by yuhan** [comment, marked as answer]
Yes, you can use [Dagster Pipes](https://docs.dagster.io/guides/dagster-pipes). It provides a `PipesK8sClient` to launch k8s pods and execute external code.

We're in the process of writing a k8s guide. Check out this draft for an early look: https://erin-pipes-k8s.dagster.dagster-docs.io/guides/dagster-pipes/integrating-kubernetes-with-dagster-pipes
